examples/extensions/xomitzero/api.yaml

```diff
@@ -0,0 +1,26 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: x-omitempty
+components:
+  schemas:
+    Client:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+        id:
+          type: number
+    ClientWithExtension:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          x-omitempty: true
+          x-omitzero: true
```

Comment:
Do both of these need to be set, here? Or should we only have `x-omitzero`?

Reply:
Sure, probably a leftover. Rebased.